app/controllers/v1/resources_controller.rb

```diff
@@ -60,6 +61,10 @@ def write_public_cocina
       File.write(File.join(purl_druid_path, 'cocina.json'), cocina_object.to_json)
     end
 
+    def write_public_purl_metadata(purl)
+      File.write(File.join(purl_druid_path, 'meta.json'), purl.as_public_json.to_json)
```

Comment:
I think we should reconsider the format of this file if we decide to go down this path.  I find that `true_targets` and `false_targets` to not be super helpful without context. So how about something like `index` and `delete`.  That's what we're using in the purl-fetcher API to create release tags: https://github.com/sul-dlss/purl_fetcher-client/blob/main/lib/purl_fetcher/client/release_tags.rb#L34

Reply:
I don't have an opinion about the format, although I do wonder if we should decouple changing the format from this change.